sklearn/calibration.py

```diff
@@ -280,16 +294,16 @@ def fit(self, X, y, sample_weight=None):
             parallel = Parallel(n_jobs=self.n_jobs)
 
             self.calibrated_classifiers_ = parallel(delayed(
-                _fit_calibrated_classifer)(clone(base_estimator),
-                                           X, y,
-                                           train=train, test=test,
-                                           method=self.method,
-                                           classes=self.classes_,
-                                           supports_sw=supports_sw,
-                                           sample_weight=sample_weight)
-                                                    for train, test
-                                                    in cv.split(X, y))
-
+                _fit_calibrated_classifer)(
```

Comment:
Let's correct the spelling:

`_fit_calibrated_classifier`

Reply:
:+1: